sct.py

```diff
@@ -269,5 +267,35 @@ def unit_tests():
     sys.exit(not result.wasSuccessful())
 
 
+@cli.command('run')
+@click.argument('argv')
+def run(argv):
```

Comment:
please add `--config` and `--backend`
hydra run-test longevity_test.LongevityTest.test_custom_time --config test-case/longevity/... --backend aws

Reply:
`hydra run-test longevity_test.LongevityTest.test_custom_time --config test-case/longevity/... --backend aws --logdir /sct`